src/Nest/XPack/Ilm/DeleteLifecycle/ElasticClient-IlmDeleteLifecycle.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Elasticsearch.Net;
+
+namespace Nest
+{
+	public partial interface IElasticClient
+	{
+		/// <summary>
+		/// Deletes a lifecycle policy.
+		/// Deletes the specified lifecycle policy definition. You cannot delete policies that are currently in use.
+		/// If the policy is being used to manage any indices, the request fails and returns an error.
+		/// </summary>
+		IIlmDeleteLifecycleResponse IlmDeleteLifecycle(PolicyId policyId, Func<IlmDeleteLifecycleDescriptor, IIlmDeleteLifecycleRequest> selector = null);
```

Comment:
Not sure about the `Ilm` prefix in the high level client; the prefix has typically not been used, except in cases where it was required to disambiguate methods. Are there any methods that would conflict if the prefix were removed?

Reply:
I've renamed everything to remove the Ilm prefix. I believe the new naming is better and should address your feedback sufficiently.